samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -87,11 +82,15 @@ static void lc3_audio_timer_timeout(struct k_work *work)
 	 */
 	const uint8_t prime_count = 2;
 	static int64_t start_time;
-	static int32_t pdu_cnt;
-	int32_t pdu_goal_cnt;
+	static int32_t sdu_cnt;
+	int32_t sdu_goal_cnt;
 	int64_t uptime, run_time_ms, run_time_100us;
+	const int frame_duration_us = bt_codec_cfg_get_frame_duration_us(audio_stream.codec);
+	const int frame_duration_100us = frame_duration_us / 100;
+	const int frames_per_sdu = bt_codec_cfg_get_blocks_per_sdu(audio_stream.codec, true);
```

Comment:
Yes and that is very intentional to highlight that the term block used in the specification refers to a frame in this context(audio).
I added a bit of clarifications to the documentation. Should I rename the function? That would make usage easier, but for those that actually read the BT specification it will blur the picture a bit. Maybe the idea is that for non-audio data you might want to use this LTV entry to describe how data is packet, but I do not know.

Reply:
> Yes and that is very intentional to highlight that the term block used in the specification

Which specification? 

Given the context (LE Audio / BAP), I think it would make the most sense to the use BAP/LE Audio terminology

Reply:
> Should I change again to frame_blocks maybe?

I was thinking the same thing :) It's more descriptive and better matches the assigned numbers terminology. The only downside is that it's a longer name for something that is already quite long.